internal/captain/values.go

```diff
@@ -155,9 +155,9 @@ func (p *PackageValue) Set(s string) error {
 	}
 	switch {
 	case strings.Contains(s, ":"):
-		v := strings.Split(s, ":")
-		p.Namespace = strings.TrimSpace(strings.Join(v[0:len(v)-1], ":"))
-		p.Name = strings.TrimSpace(v[len(v)-1])
+		namespace, name, _ := strings.Cut(s, ":")
```

Comment:
With Java, we're doing things like `state install language/java:junit:junit` and `state uninstall language/java:junit:junit`, so we want to split the string only once on the first ':' and keep 'junit:junit' intact.

Reply:
Oh joy! I really wish we could think of a better way to represent this. Not for this PR, but starting to prefer just passing a flag at this point, eg. `state install -l java junit:junit`. Or maybe even make it a required argument; `state install java junit:junit`.